specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json

```diff
@@ -0,0 +1,5445 @@
+      "properties": {
+        "filters": {
+          "$ref": "#/definitions/BlobInventoryPolicyFilter",
+          "description": "An object that defines the filter set."
+        },
+        "format": {
+          "type": "string",
+          "description": "This is a required field, it specifies the format for the inventory files.",
+          "enum": [
+            "Csv",
+            "Parquet"
+          ],
+          "x-ms-enum": {
+            "name": "format",
+            "modelAsString": true
+          }
+        },
+        "schedule": {
+          "type": "string",
+          "description": "This is a required field. This field is used to schedule an inventory formation.",
+          "enum": [
+            "Daily",
+            "Weekly"
+          ],
+          "x-ms-enum": {
+            "name": "schedule",
+            "modelAsString": true
+          }
+        },
+        "objectType": {
+          "type": "string",
+          "description": "This is a required field. This field specifies the scope of the inventory created either at the blob or container level.",
+          "enum": [
+            "Blob",
+            "Container"
+          ],
+          "x-ms-enum": {
+            "name": "objectType",
+            "modelAsString": true
+          }
+        },
+        "schemaFields": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "This is a required field. This field specifies the fields and properties of the object to be included in the inventory. The Schema field value 'Name' is always required. The valid values for this field for the 'Blob' definition.objectType include 'Name, Creation-Time, Last-Modified, Content-Length, Content-MD5, BlobType, AccessTier, AccessTierChangeTime, AccessTierInferred, Tags, Expiry-Time, hdi_isfolder, Owner, Group, Permissions, Acl, Snapshot, VersionId, IsCurrentVersion, Metadata, LastAccessTime, Tags, Etag, ContentType, ContentEncoding, ContentLanguage, ContentCRC64, CacheControl, ContentDisposition, LeaseStatus, LeaseState, LeaseDuration, ServerEncrypted, Deleted, DeletionId, DeletedTime, RemainingRetentionDays, ImmutabilityPolicyUntilDate, ImmutabilityPolicyMode, LegalHold, CopyId, CopyStatus, CopySource, CopyProgress, CopyCompletionTime, CopyStatusDescription, CustomerProvidedKeySha256, RehydratePriority, ArchiveStatus, XmsBlobSequenceNumber, EncryptionScope, IncrementalCopy, TagCount'. The valid values for 'Container' definition.objectType include 'Name, Last-Modified, Metadata, LeaseStatus, LeaseState, LeaseDuration, PublicAccess, HasImmutabilityPolicy, HasLegalHold, Etag, DefaultEncryptionScope, DenyEncryptionScopeOverride, ImmutableStorageWithVersioningEnabled, Deleted, Version, DeletedTime, RemainingRetentionDays'. Schema field values 'Expiry-Time, hdi_isfolder, Owner, Group, Permissions, Acl' are valid only for Hns enabled accounts.'Tags, DeletionId, DeletedTime' field is only valid for Non-Hns accounts."
```

Comment:
This line says 
> 'Tags, DeletionId, DeletedTime' field is only valid for Non-Hns accounts.

But per [doc1](https://microsoft.sharepoint.com/:w:/t/AzureStorage/EYSTBDpPnfVPjuPsqEQ4Om0BJCeDSxQq-uy7FKZ2EDnFTQ?e=htRJ2U), [doc2](https://microsoft.sharepoint.com/:w:/t/AzureStorage/EcOq63q0mttDoB25ZeguIsEBwbufeSCh7wmwE5INovGFZg?e=tl7ocg&CID=9F618CE7-2108-4715-837F-2ED4A2DAC2A2&wdLOR=cAA4B65C3-9374-4052-9F3D-D3E059B0BED4):
1. DeletionId, DeletedTime only for HNS enabled account
2. TagCount only valid for Non-Hns accounts.

Reply:
Confirmed my below statements from IDC backend team for BlobInventory
For Container object Type Mandatory and Legal fields for FNS - Deleted, Version, DeletedTime, RemainingRetentionDays
For Container object Type Mandatory and Legal fields for HNS- Deleted, Version, DeletedTime, RemainingRetentionDays

For Blob object Type Mandatory and Legal Fields for FNS - Deleted, RemainingRetentionDays
For Blob object Type Mandatory and Legal Fields for HNS - DeletionId, Deleted, DeletedTime, RemainingRetentionDays.

Yes, TagCount is only for Non-Hns accounts, i have updated the description and the server. We have not enabled this anywhere other than canary we will start enabling it slowly after this small change reaches all regions.